packages/component-library/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@shopware-ag/meteor-component-library",
```

Comment:
@jleifeld could you check what's correct here?

Reply:
What do you mean exactly?